azurelinuxagent/common/event.py

```diff
@@ -37,6 +37,8 @@
 
 _EVENT_MSG = "Event: name={0}, op={1}, message={2}, duration={3}"
 TELEMETRY_EVENT_PROVIDER_ID = "69B669B9-4AF8-4C50-BDC4-6006FA76E975"
+TELEMETRY_METRICS_PROVIDER_ID = "69B669B9-4AF8-4C50-BDC4-6006FA76E975"
```

Comment:
Because they are sending to two different ids (and in the future, we might send it to different locations (perf and events at different places). But yes, I can push that task into the future and remove the redundant constant.